### ISSUE TITLE
Calculation of accuracy

### ISSUE THREAD

**Issue by XALEX-123** [beuve/DmyT#2]
```
         else:
            model.train()
            optimizer.zero_grad()
            output = model.forward(data)
            loss = criterion(output, target.argmax(dim=1))
            loss.backward()
            if criterion.name == 'DmyT':
                output = get_dummy_prediction(output, criterion._loss.dummies,
                                              device)
            elif criterion.name == 'Triplet':
                output = torch.zeros((target.size(0))).to(device)
            else:
                output = output.argmax(dim=1)
            optimizer.step()

        accuracy = (output == target.argmax(dim=1)).float().mean()
```
        
Why is torch.zeros used in the output of Triplet? Is the accuracy calculated this way meaningful?

**Comment by beuve** [comment]
I couldn't compute an accuracy from the triplet loss output since there is no prediction. The `torch.zeros` only ensures that the last line (`accuracy = (output == target.argmax(dim=1)).float().mean()`) doesn't return an error, but the displayed accuracy is meaningless.
